crates/bevy_gltf/src/loader.rs

```diff
@@ -697,7 +697,7 @@ fn load_node(
     gltf_node: &gltf::Node,
     world_builder: &mut WorldChildBuilder,
     load_context: &mut LoadContext,
-    buffer_data: &[Vec<u8>],
+    _buffer_data: &[Vec<u8>],
```

Comment:
Why was this renamed? The commit message is ` buffer_data renamed to _buffer_data as it only was used in recursion`, but I don't think this is the appropriate action.

Prepending an _ is used for surprising the unused variable lint, not to denote a recursion-relevant variable.
If the lint is actually triggered due to the changes, I would recommend an `#[cfg_attr(feature = "serde", allow(unused-assignments]))]` (I think that's the correct name of the lint here)

Reply:
Okay, I will explore this!

Reply:
Btw, should I place a ``#[cfg_attr(feature = "serde", allow(unused-assignments]))]`` in the ``loader`` module? Should I then also connect the ``serde`` feature to the ``bevy_gltf`` crate?

Or did you have something else in mind?

Reply:
>  Should I then also connect the serde feature to the bevy_gltf crate?
There should be no need for this.

Consider just reverting the change. Afaik it shouldn't be necessary.